cloud-minecraft/cloud-bungee/src/main/java/cloud/commandframework/bungee/BungeeCommandManager.java

```diff
@@ -46,21 +57,33 @@
      * @param commandExecutionCoordinator  Coordinator provider
      * @param commandSenderMapper          Function that maps {@link CommandSender} to the command sender type
      * @param backwardsCommandSenderMapper Function that maps the command sender type to {@link CommandSender}
-     * @throws Exception If the construction of the manager fails
      */
     public BungeeCommandManager(
             final @NonNull Plugin owningPlugin,
             final @NonNull Function<@NonNull CommandTree<C>,
                     @NonNull CommandExecutionCoordinator<C>> commandExecutionCoordinator,
             final @NonNull Function<@NonNull CommandSender, @NonNull C> commandSenderMapper,
             final @NonNull Function<@NonNull C, @NonNull CommandSender> backwardsCommandSenderMapper
-    )
-            throws Exception {
+    ) {
         super(commandExecutionCoordinator, new BungeePluginRegistrationHandler<>());
         ((BungeePluginRegistrationHandler<C>) this.getCommandRegistrationHandler()).initialize(this);
         this.owningPlugin = owningPlugin;
         this.commandSenderMapper = commandSenderMapper;
         this.backwardsCommandSenderMapper = backwardsCommandSenderMapper;
+
```

Comment:
The PlayerArgument and ServerArgument parsers should be registered with the parser registry here, similar to [this](https://github.com/Incendo/cloud/commit/4995b4c2d7de476b9d23c1c97cd3751bc75550b6) commit

Reply:
Added, working on fixing the branch now.